### PR TITLE
fix(service-worker): update to fix important issue on FF and Chrome

### LINKED ISSUE #199: Update to sw-precache ^5.0.0 for Chrome 59 and latest Firefox support
### Description
https://github.com/GoogleChrome/sw-precache/pull/241 includes an important fix for Chrome M59 which is expected to break a lot of apps using old versions of sw-precache in June 2017. Firefox has already disabled this redirect behavior which may cause issues for apps using the existing sw-precache from polymer-build and polymer-cli.

### Versions & Environment
- polymer-build: v0.13.0 and v0.17.0
- node: 6.10.2
- Operating System: OS X El Capitan

#### Steps to Reproduce
1. Follow the steps listed here: https://www.polymer-project.org/1.0/toolbox/service-worker
1. The load your site with Chrome Version 58.0.3029.96 (64-bit) or similar.
1. Press the reload button

#### Expected Results
The page reloads nearly instantly w/o no errors in the console related to sw-precache.

#### Actual Results
In Chrome 59, the navigation to "https://devfestflorida.org/" will result in a network error, because `FetchEvent.respondWith()` was called with a redirected response. See https://crbug.com/658249. The url list of the response was: `["https://devfestflorida.org/index.html?sw-precache=d29f0089363aaf48cd80c928c9628029", "https://devfestflorida.org/?sw-precache=d29f0089363aaf48cd80c928c9628029"]`

[Deprecation] The service worker responded to the navigation request with a redirected response. This will result in an error in M59, around June 2017.

We noticed this as part of https://github.com/DevFestFlorida/DevFestFloridaSite/issues/36 and the GDG-X Hoverboard project has actually moved to a custom Polymer build (https://github.com/gdg-x/hoverboard/pull/235) instead of using polymer-cli so that they can load the latest (https://github.com/gdg-x/hoverboard/commit/d75b2d392f2464c16d7c9e84c0ef586e88be4b14) sw-precache ^5.0.0.

### PR DESCRIPTION
**Pull request by davinkevin**
Link to https://github.com/gdg-x/hoverboard/issues/253 and https://github.com/GDGToulouse/site-devfest-toulouse-2017/issues/23 (and many more GDG website)

Closes #199

 - [x] CHANGELOG.md has been updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "polymer-analyzer": "2.0.0-alpha.40",
     "polymer-bundler": "2.0.0-pre.14",
     "polymer-project-config": "^2.0.0",
-    "sw-precache": "^4.2.0",
+    "sw-precache": "^5.0.0",
```
Comment (ozasadnyy) [inline package.json]: 5.1.1 is the latest version:
https://github.com/GoogleChrome/sw-precache/releases/tag/5.1.1

**Comment by FredKSchott** [comment]
sw-precache CHANGELOG here: https://github.com/GoogleChrome/sw-precache/releases

It looks like I was wrong and there weren't any breaking interface changes, which is good. Can you add a note to our changelog that service worker caches will be repopulated when users update polymer-build? Adding a pointer to that release summary would be good as well.

File: package.json (hunk 1 above)
Comment (FredKSchott) [inline package.json]: ^5.0.0 will get 5.1.1 and higher 5's, but it looks like there are some bug fixes in those later versions that we specifically want to include. +1 on changing this to `^5.1.1`

**Comment by davinkevin** [comment]
CHANGELOG.md updated and version align to your request, `^5.1.1`

Thanks

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased 
+
+* Dependency updates. Upgraded to `sw-precache@5.1.1` minimum to prevent error about corrupted data on firefox 52 and Chrome 60 (witch match the spec)
```
Comment (ozasadnyy) [inline CHANGELOG.md]: As far as I know it doesn't work on Chrome >= 59.
Firefox >=50

Comment (stramel) [inline CHANGELOG.md]: `witch` -> `which` and `match` -> `matches`

Comment (Splaktar) [inline CHANGELOG.md]: I believe that this should be Chrome 59. Probably want to capitalize Firefox. Did you mean 'which match the spec'?

Comment (stramel) [inline CHANGELOG.md]: :+1: for Chrome 59

Comment (davinkevin) [inline CHANGELOG.md]: I should not do PR from my tablet in my bed around midnight...  😄 
I do a new modification...

Comment (Splaktar) [inline CHANGELOG.md]: @davinkevin Hey, no problem! Thanks for getting this PR kicked off!

**Pull request merged**